### PR TITLE
Add Vietnamese translations for reply messages

### PR DESCRIPTION
**Pull request by TamamoNoMae13**
`tosurnament/tournament` is partially translated (didn't translate the part that post_result requires browsing the wiki for params)
`tosurnament/bracket` is ignored.

### CODE REVIEW

File: backend/bot/replies/tosurnament/module/vi.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "not_a_player": "Tôi không thể tìm bạn có phải là một người chơi, lý do có thể là:\n- Bạn đã liên kết tài khoản osu! với bot nhưng chưa thực hiện lệnh `%0name_change`\n- Thẻ tên Discord của bạn đã thay đổi (kiểm tra cả số định danh nữa), với trường hợp này, bạn hãy thử liên kết tài khoản của bạn với lệnh `%0link`\n",
+  "invalid_match_id_or_no_bracket_role": "ID trận không hợp lệ hoặc thiếu vai trò bảng đấu được yêu cầu.",
+  "invalid_match_id": "ID trận không hợp lệ.",
+  "no_challonge": "Không có challonge được thiết lập cho bảng đấu `%1`.",
+  "challonge_no_rights": "Bạn không có quyền viết vào challonge.",
+  "challonge_not_found": "Không tìm thấy challonge.",
+  "invalid_mp_link": "Đường dẫn mp không hợp lệ.",
+  "participant_not_found": "Không thể cập nhật trận %1 trên challonge. Hãy đảm bảo tên của những người tham gia trên challonge đã chính xác.",
+  "match_not_found": "Không tìm thấy trận, có vẻ như có lỗi gì đó trên challonge.",
+  "spreadsheet_rights": "Ôi không, hình như tôi không có quyền %1 trên Trang tính %3 của bảng đấu %2."
```
Comment (SpartanPlume) [inline backend/bot/replies/tosurnament/module/vi.json:27]: this one has been modified recently, so an update is needed

File: backend/bot/replies/module/vi.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "tosurnament_api_error": "Một lỗi xảy ra:\nMã: `%1`\nMô tả: `%2`",
+  "usage": "Sử dụng",
+  "example": "Ví dụ",
+  "none": "Không có",
+  "match": "Trận đấu",
+  "referee": "Trọng tài",
+  "streamer": "Người phát trực tiếp",
+  "commentator": "Bình luận viên",
+  "undefined": "Không xác định"
+}
```
Comment (SpartanPlume) [inline backend/bot/replies/module/vi.json:27]: 2 new fields have been added (used in get_matches and show_next_matches commands): `qualifier` and `teams`

**Review by SpartanPlume — CHANGES_REQUESTED** [review]
It's fine to not do the bracket folder and the post_result replies as the bracket settings will be done via a web interface soon, and the post_result module will be completely refactored.

Just 2 comments because I did some updates while you were translating these and we're good to go.

**Comment by TamamoNoMae13** [comment]
I've changed things according to your request. 👌🏼

**Review by SpartanPlume — APPROVED** [review]

**Comment by SpartanPlume** [comment]
Bot updated with the new language, thanks for the contribution. =)

**Pull request merged**